lasagne/layers/embedding.py

```diff
@@ -18,7 +18,7 @@ class EmbeddingLayer(Layer):
     Parameters
     ----------
     incoming : a :class:`Layer` instance or a tuple
-        The layer feeding into this layer, or the expected input shape
+        The layer feeding into this layer, or the expected input shape.
```

Comment:
The current docs are rife with inconsistencies like this one. It bothers me personally because I'm a bit of a perfectionist in that sense, but objectively it's really not that important right now. We should focus on the big things and not spend all of our time 'cleaning things up' right now, if possible :)